nerdlets/groundskeeper-v2/hooks/useFetchEntitiesDetails.js

```diff
@@ -5,12 +5,33 @@ import { APPS_DETAILS } from '../queries';
 import { AGENTS, AGENTS_REGEX_STRING, RUNTIMES } from '../constants';
 
 const MAX_ENTITIES_IN_SET = 10;
+
+/*distributedTracingEnabledValuesByLanguage = {
+  "go":"DistributedTracer.Enabled",
+  "java":"distributed_tracing.enabled",
+  "dotnet":"distributed_tracing.enabled",
+  "nodejs":"distributed_tracing.enabled",
+  "php":"newrelic.distributed_tracing_enabled",
+  "python":"distributed_tracing.enabled",
+  "ruby":"distributed_tracing.enabled"
+};
+logsInContextEnabledValuesByLanguage = {
+  "go":"ApplicationLogging.Enabled",
+  "java":"distributed_tracing.enabled",
+  "dotnet":"distributed_tracing.enabled",
+  "nodejs":"distributed_tracing.enabled",
+  "php":"newrelic.distributed_tracing_enabled",
+  "python":"distributed_tracing.enabled",
+  "ruby":"distributed_tracing.enabled"
+*/
 const featuresList = {
-  dtEnabled: 'newrelic.distributed_tracing.enabled',
+  dtEnabled: ['newrelic.distributed_tracing.enabled', 'newrelic.newrelic.distributed_tracing_enabled', 'newrelic.DistributedTracer.Enabled'],
   infTraceHost: 'newrelic.infinite_tracing.trace_observer_host',
   logEnabled: 'newrelic.application_logging.enabled'
```

Comment:
Shouldn't the possible values for logs in context be listed as array here?

Reply:
Building on this, I recommend we turn all three of them into arrays, even if `infTraceHost` may only have one value. That way if new values are added in the future, we just need to add them to the appropriate array.

Reply:
Done - Had git issues so submitting a new PR :)